packages/types/README.md

```diff
@@ -1,23 +1,165 @@
+| [[DigestItem]] | A [[EnumType]] the specifies the specific item in the logs of a [[Digest]] |
+| [[Event]] | Wrapper for the actual data that forms part of an [[Event]] |
+| [[EventRecord]] | A record for an [[Event]] (as specified by [[Metadata]]) with the specific [[Phase]] of application |
+| [[Exposure]] | A snapshot of the stake backing a single validator in the system |
+| [[Extrinsic]] | Representation of an Extrinsic in the system |
+| [[ExtrinsicEra]] | The era for an extrinsic, indicating either a mortal or immortal extrinsic |
+| [[Extrinsics]] | A [[Vector]] of [[Extrinsic]] |
+| [[ExtrinsicSignature]] | A container for the [[Signature]] associated with a specific [[Extrinsic]] |
+| [[Gas]] | A gas number type for Substrate, extending [[U64]] |
+| [[Hash]] | The default hash that is used accross the system. It is just a thin wrapper around [[H256]] |
+| [[Header]] | A [[Block]] header |
+| [[HeaderExtended]] | A [[Block]] header with an additional `author` field that indicates the block author] |
+| [[IndividualExposure]] | The Substrate IndividualExposure for staking |
+| [[InherentOfflineReport]] | Describes the offline-reporting extrinsic |
+| [[Justification]] | A generic justification as a stream of [[Bytes]], this is specific per consensus implementation |
+| [[Key]] | The Substrate Key representation as a [[Bytes]] (`vec<u8>`) |
+| [[KeyValue]] |  KeyValue structure. Since most of the keys and resultant values in Subtrate are hashed and/or encoded, keys and values are reprsented as [[Bytes]] |
+| [[KeyValueOption]] | A key/value change. Similar to the [[KeyValue]] structure, but the value can be optional |
+| [[LockIdentifier]] | The Substrate LockIdentifier for staking |
+| [[LocKPeriods]] | A number of lock periods |
+| [[MisbehaviorKind]] | An [[EnumType]] containing a Bft misbehaviour |
+| [[MisbehaviorReport]] | A Misbehaviour report of [[MisbehavioirKind]] against a specific [[AuthorityId]] |
+| [[NewAccountOutcome]] | Enum to track the outcome for creation of an [[AccountId]] |
+| [[NextAuthority]] | The next authority available as [[SessionKey]] |
+| [[Nonce]] | The Nonce or number of transactions sent by a specific account |
+| [[NonceCompact]] | The Compact<Nonce> or number of transactions sent by a specific account |
+| [[Origin]] | Where Origin occurs, it should be ignored as an internal-only value |
+| [[ParaId]] | Identifier for a deployed parachain implemented as a [[U32]] |
+| [[Perbill]] | Parts per billion (see also [[Permill]]) |
+| [[Permill]] | Parts per million (See also [[Perbill]]) |
+| [[PrefabWasmModule]] | Struct to encode the vesting schedule of an individual account |
+| [[PropIndex]] | An increasing number that represents a specific council proposal index in the system |
+| [[Proposal]] | A proposal in the system. It just extends [[Method]] (Proposal = Call in Rust) |
+| [[ProposalIndex]] | An increasing number that represents a specific council proposal index in the system |
```

Comment:
`PropIndex` and `ProposalIndex` seem to do the exact same thing. Is there a reason to have both or can we delete one of the two?

Reply:
One is in democracy, the other in treasury, so unfortunately we need them both for now

Reply:
Ah, ok. It just looked like an oversight, but that explains it.